metadeploy/api/push.py

```diff
@@ -150,3 +158,46 @@ async def notify_org_result_changed(result):
     if await get_set_message_semaphore(channel_layer, message):
         logger.info(f"Sending message {message}")
         await channel_layer.group_send(group_name, message)
+
+
+async def notify_org_finished(scratch_org, error=None):
```

Comment:
Is this actually being used outside of tests?

Reply:
Yep, in `models.ScratchOrg.fail` and `.complete`.